examples/07-Train-a-third-party-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Can we rename it to Training a XGBoost model - similar to the other headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/629/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
done